dub.json

```diff
@@ -15,6 +15,8 @@
     "sourceFiles": [
         "source/scpp/build/BallotProtocol.o",
         "source/scpp/build/ByteSliceHasher.o",
+        "source/scpp/build/DSizeChecks.o",
+        "source/scpp/build/DLayoutChecks.o",
```

Comment:
Can we move this in tests only ?
See https://dub.pm/package-format-json.html#configurations

Reply:
Linking to the dub docs is cruel.